src/Compilers/CSharp/Test/Syntax/Parsing/TopLevelStatementsParsingTests.cs

```diff
@@ -3597,5 +3597,426 @@ partial ext X
             }
             EOF();
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/67311")]
+        public void DelegateWithTupleReturnType_TopLevel()
```

Comment:
>DelegateWithTupleReturnType_TopLevel [](http://example.com/codeflow?start=20&length=36)

It looks like these tests belong in `DeclarationParsingTests.cs`. This file is for top level statements feature.